cluster-autoscaler/core/utils/utils.go

```diff
@@ -212,3 +213,26 @@ func GetOldestCreateTimeWithGpu(pods []*apiv1.Pod) (bool, time.Time) {
 	}
 	return gpuFound, oldest
 }
+
+// FilterRecreatablePods filters pods that will be recreated by their controllers
+func FilterRecreatablePods(pods []*apiv1.Pod) []*apiv1.Pod {
```

Comment:
Would it be possible to move these functions to a place with a name that is less generic than "utils"?